### DISCUSSION TITLE
Arrange the result

### DISCUSSION THREAD

**Discussion by adumith** [microsoft-search/pnp-modern-search#2039] [Q&A]
Hello everyone,

I just start to play around with this package, so I have many question about this, for example this one.

The web part is show me the results in vertical way and just the photo, Full name, Job Title and email.

I wondering if the is a way to arrange to set up in horizontal way and an small card version, like this one:
[https://ibb.co/Y0H4r8K](url)

Thank in advance,

ADumith

**Comment by wobba** [comment]
Hi, you can use any of the default templates and customize them, or use a custom handlebar template with the CSS and HTML which is what you want/need. https://microsoft-search.github.io/pnp-modern-search/usage/search-results/layouts/ (And the image link is not working.)

**Reply by adumith** [reply]
Hello!

Thank you for replying, however I must confess I was expecting a more collaborative response (maybe), the documentation frankly doesn't say much about the potential of this web part for employee directory.

I have searched on YouTube and it's as nobody uses it for that, could it be because it doesn't work or that this functionality has not been exploited?

It seems I have to keep walking blind, 

Thanks any way

This is the image
![vcardIntranet](https://user-images.githubusercontent.com/101891060/165384320-8ce53239-b581-4a14-8b15-96aa6ea1a4d7.png)

**Reply by FranckyC** [reply]
Hi @adumith, we don't have a tutorial for this scenario in the PnP Modern Search Web Part but you can still look here https://aequos-solutions.github.io/modern-data-visualizer/getting_started/tutorials/build_people_directory_sharepoint_search/. This is basically the same configuration. For the UI part, it is up to you to build it using Handlebars template.

**Comment by wobba** [comment]
We just got a PR for a people directory https://github.com/microsoft-search/pnp-modern-search/pull/2043. See docs at https://github.com/microsoft-search/pnp-modern-search/blob/develop/docs/scenarios/Create-a-useful-People-Search.md until they are published.
